runtime/src/validate_block/implementation.rs

```diff
@@ -56,14 +57,14 @@ trait Storage {
 #[cfg(not(feature = "std"))]
 #[doc(hidden)]
 pub fn validate_block<B: BlockT, E: ExecuteBlock<B>>(
-	mut block_data: &[u8],
+	mut arguments: &[u8],
 ) {
-	use codec::Decode;
```

Comment:
Please bring this back and remove from the top of the file.

Did you tried to compile this for wasm and std?